thread-sanitizer-ignorelist.txt

```diff
@@ -11,6 +11,3 @@ signal:*
 
 # TODO: Remove: There's something weird going on with MPI code I don't understand
 race:faabric::scheduler::MpiWorld::*
-
```

Comment:
I also remove this from here as discussed offline with jakub (don't think it is worth tagging them here).